ui/HomePage.py

```
@@ -0,0 +1,35 @@
+    st.write("DataFrame:", dataframe)
+
+    # Convert DataFrame to JSON
+    json_data = dataframe.to_json(orient="records")  # Convert to JSON format
+
+    # Parse JSON string for pretty printing
+    parsed_json = json.loads(json_data)
+    
+    # Display JSON
+    st.write("JSON Data:", json.dumps(parsed_json, indent=4))
+
+result = st.button("RUN");
```

Ruff

Statement ends with an unnecessary semicolon

Remove unnecessary semicolon

(E703)